generator/index.js

```diff
@@ -142,6 +142,13 @@ module.exports = async (api, options, rootOptions) => {
       fs.ensureFileSync(api.resolve(path.join(genConfig.dirPathPrefix, 'babel.config.js')));
       await applyBabelConfig(api, genConfig, 'babel.config.js');
     });
+
+    api.render(async () => {
+      if (options.templateType == 'custom') {
```

Comment:
I think better to check `options.customTemplatePath` is exist.

Reply:
done